app/components/dashboard/show/collections_list_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render Elements::HeaderComponent.new('h3', classes: ['mb-4'], value: label) %>
+
+<% collections.each do |collection| %>
+  <%= render Elements::HeaderComponent.new('h4', value: link_to(collection.title, "/collections/#{collection.druid}")) %>
+  <%= render Dashboard::Show::WorksListComponent.new(collection:) %>
+
+  <%= link_to new_work_path, class: 'btn btn-primary' do %>
+    <div class="align-self-center d-flex justify-content-center">Deposit to this collection</div>
```

Comment:
Why does the label need to be in a div?

Reply:
Cruft from H2 I think, this goes away when using the `ButtonLinkComponent`